api/v1alpha1/dspipeline_types.go

```diff
@@ -158,18 +182,22 @@ type ExternalDB struct {
 type ObjectStorage struct {
 	*Minio           `json:"minio,omitempty"`
 	*ExternalStorage `json:"externalStorage,omitempty"`
+	// Default: false
 	// +kubebuilder:default:=false
 	// +kubebuilder:validation:Optional
 	DisableHealthCheck bool `json:"disableHealthCheck"`
 }
 
 type Minio struct {
+	// Default: true
 	// +kubebuilder:default:=true
 	// +kubebuilder:validation:Optional
 	Deploy bool `json:"deploy"`
```

Comment:
"Enable DS Pipelines Operator management of Minio. Setting Deploy to false disables operator reconciliation."